src/Corax/Querying/IndexSearcher.TermMatch.cs

```diff
@@ -234,16 +234,19 @@ private long NumberOfDocumentsUnderSpecificTerm(in FieldMetadata binding, string
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     internal long NumberOfDocumentsUnderSpecificTerm(CompactTree tree, Slice term)
     {
-        return NumberOfDocumentsUnderSpecificTerm(tree, term.AsReadOnlySpan());
-    }
-
-    [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    private long NumberOfDocumentsUnderSpecificTerm(CompactTree tree, ReadOnlySpan<byte> term)
-    {
-        if (tree.TryGetValue(term, out var value) == false)
+        var termAsSpan = term.AsReadOnlySpan();
+        if (tree.TryGetValue(termAsSpan, out long containerId) == false)
+        {
+            if (termAsSpan is [0])
```

Comment:
What does this mean? How are we getting this value?

Reply:
If we have a `null` term we want to get number of documents from posting list that handles nulls rather than attempt to get it from the tree.

Such term is passed here as `Constants.NullValueSlice`, which corresponds to a span with single `0` byte.

Reply:
Let's introduce a constant then. You will be able to use it here then:
https://github.com/ravendb/ravendb/blob/98db5c3a0745350227aa53ab26b32cdca82c5d89/src/Corax/Constants.cs#L15

Reply:
Added